src/components/ebay-textbox/test/test.server.js

```diff
@@ -0,0 +1,32 @@
+
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const rootSelector = `span.textbox`;
+const inputSelector = `input.textbox__control`;
+const textareaSelector = `textarea.textbox__control`;
+
+test('renders default input textbox', context => {
+    const $ = testUtils.getCheerio(context.render());
+    expect($(rootSelector).length).to.equal(1);
+    expect($(inputSelector).length).to.equal(1);
+});
+
+test('renders a input textbox with invalid/error state', context => {
+    const input = { invalid: true };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect(Boolean($(inputSelector).prop('aria-invalid'))).to.equal(true);
+});
+
+test('renders a textarea element', context => {
+    const input = { multiline: true };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($(textareaSelector).length).to.equal(1);
+});
+
```

Comment:
Would be great if you could also cover the `fluid` use case so that this code is covered: https://coveralls.io/builds/18237755/source?filename=src%2Fcomponents%2Febay-textbox%2Findex.js#L8

Reply:
Done